### ISSUE TITLE
Crashes on loading Session.spfs file

### ISSUE THREAD

**Issue by SBerkovic** [Sequel-Ace/Sequel-Ace#1019]
- Sequel Ace Version (including build number): 3.3.0 (3026)
- Sequel Ace Source (App Store/GitHub/Homebrew): App Store
- macOS Version: 11.2.3 (20D91)
- Processor Type (Intel/Apple): Intel
- MySQL Version: 5.6
- macOS Localization: English US

**Is Issue Present in [Latest Beta](https://github.com/Sequel-Ace/Sequel-Ace/releases)? Yes

**Description**
Sequel Ace crashes when trying to load Session.spfs file

**Steps To Reproduce**
1. open session file with sequel-ace 

**Expected Behavior**
Sequel Ace opens my saved session with queries and connection tabs open

**Related Issues**

**Additional Context**
downgrading to version 3.2.3 (3024) (installed via github) and loading session works fine

**Comment by Kaspik** [comment]
Hmm, I'm not able to repro crash, but there are some issue with import of the session (tabs are not loaded as tabs, but windows). Not sure how to track the crash down...

**Comment by Jason-Morcos** [comment]
@SBerkovic Please upload the crash file here so we can trace the issue. You can find crash reports in the Console.app

**Comment by SBerkovic** [comment]
> @SBerkovic Please upload the crash file here so we can trace the issue. You can find crash reports in the Console.app
Here's the crash file

[Sequel Ace_2021-04-08-132645.crash.zip](https://github.com/Sequel-Ace/Sequel-Ace/files/6280424/Sequel.Ace_2021-04-08-132645.crash.zip)

**Comment by Kaspik** [comment]
`Attempted to connect a connection that is not disconnected (SPMySQLConnectionState=1).` - That's related to that freakin MySQL connection that somehow tries to connect even tho it's already connected.... no clue from my side, I'm just ready to completely wipe of MySQL connection and write it new in Swift.... 🤦🏼‍♂️

**Comment by SBerkovic** [comment]
now its crashing when i attempt to connect
```
Application Specific Information:
Fatal error: Expected superview but found nil when attempting make constraint `equalToSuperview`.: file SnapKit/ConstraintMakerRelatable.swift, line 85
```

Should I upload the whole file?

**Comment by SBerkovic** [comment]
this big is still present in Version 3.3.2 Beta Build 3029

**Comment by SBerkovic** [comment]
this bug is still present in Version 3.3.3 Beta Build 3032

**Comment by SBerkovic** [comment]
On latest beta - 3.5.0 (20028) Beta 2, it's not crashing but instead of opening tabs, it opens separate windows. Also, the query editor content is not being loaded from the session spfs file.

**Comment by Kaspik** [comment]
This has been fixed for 4.0.0 Beta that will go out.
